benchmarks/bulk-insert-and-query.cc

```diff
@@ -344,11 +344,8 @@ int main(int argc, char * argv[]) {
   std::map<int,std::string> names = {
     // Xor
     {1000, "Xor8naive"}, {1002, "Xor16naive"},
-    {2000, "Xor8prefetch"}, {2002, "Xor16prefetch"},
     {0, "Xor8"}, {1, "Xor12"}, {2, "Xor16"},
     {3, "Xor+8"}, {4, "Xor+16"},
-    {5, "Xor10"}, {6, "Xor10.666"},
```

Comment:
I would actually keep them, because they have been referenced in the fuse filter paper (I think).